### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by mspiekermann** [eclipse-edc/Connector#82] [Announcements]
## 👋 Welcome to the Eclipse Dataspace Connector!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by juliapampus** [comment]
To get an introduction to this project and an overview of important facts and links, please take a look at our [GitHub page](https://eclipse-dataspaceconnector.github.io/DataSpaceConnector/#/README).

**Reply by gtoubekis** [reply]
> To get an introduction to this project and an overview of important facts and links, please take a look at our [GitHub page](https://eclipse-dataspaceconnector.github.io/DataSpaceConnector/#/README).

Hi folks,
though a helpful hint - the link unfortunately is outdated and I assume it should read <https://github.com/eclipse-edc>
